.buildkite/pipeline.yml

```
@@ -77,57 +77,57 @@ steps:
     key: "gpu_jra55_tests"
     env:
       TEST_GROUP: "jra55"
+      GPU_TEST: "true"
```

A bit more flexible would be something like

```suggestion
      OCEANANIGANS_ARCHITECTURE: "GPU"
```

I suppose "false" is meant to signify CPU. But explicitly denoting the architecture might allow additional possibilities.